nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/DomainDiscoveryService.java

```diff
@@ -53,45 +51,31 @@
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
 import java.net.UnknownHostException;
-import java.util.Collections;
 import java.util.LinkedList;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 import java.util.Properties;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import org.glassfish.api.admin.ServerEnvironment;
 import org.glassfish.grizzly.utils.Holder;
 import org.glassfish.internal.api.Globals;
-import org.glassfish.internal.api.ServerContext;
 
 /**
  * Provides a Discovery SPI implementation for Hazelcast that uses knowledge
  * of the domain topology to build out the cluster and discover members
  * @since 5.0
  * @author steve
  */
-public class DomainDiscoveryService implements DiscoveryService {
+public class DomainDiscoveryService {
```

Comment:
This somewhat feels like a dead class now - it could just be rolled into the new DomainDiscoveryStrategy as that's the only palce it's used (other than any pre-configured logger config).

At the very least the class description is now inaccurate - it's not an SPI implementation anymore.

Reply:
yea makes sense

Reply:
done